RecoPPS/RPixEfficiencyTools/README.md

```diff
@@ -8,4 +52,57 @@ Instruction on how to setup and run efficiency suite in CMSSW_11_3_2 environment
 7. Create directories for files used during the run: mkdir InputFiles OutputFiles Jobs LogFiles
 8. Prepare the input .dat file for the chosen era with <era name>. For instance, you can specify this file to load all the input .root files from the chosen directory by typing:
 	ls /path/to/your/input/files/*.root | sed 's/^/file:/' > InputFiles/Era<era name>.dat
-9. bash submitEfficiencyAnalysisEra.sh <era name>
\ No newline at end of file
+9. *(probably outdated to delete)* bash submitEfficiencyAnalysisEra.sh <era name>
+
+
+## How to test Efficiency Analyis and Reference Analysis 
+1. Set up CMSSW_12_4_0 project (follow steps 1-7 of `Project Setup` section) just checkout different branch
+	1. *(STEP 4 modification)* `git cms-merge-topic matt628:automation/for_CMSSW12-update-DQMWorker`
+	2. Remember to run cmsenv in each terminal session at src directory
+	3. Remember to create needed directories:
+		```mkdir InputFiles OutputFiles Jobs LogFiles```
+2. Prepare test file and test setup:
+	1. **file** *(note/TODO: this file makes sense for EA, but doesn't make sense for RA please find a better file )* `/eos/project-c/ctpps/subsystems/Pixel/RPixTracking/EfficiencyCalculation2018/ReRecoEras_CMSSW_10_6_10/EGamma/re-MINIAOD/ctpps_2018A_pixelefficiency_singleele200630_073503/0000/reMINIAOD_A.root`
+	2. **file list to analyse** Just create .dat file with paths you want to analyse.
+	```ls /eos/project-c/ctpps/subsystems/Pixel/RPixTracking/EfficiencyCalculation2018/ReRecoEras_CMSSW_10_6_10/EGamma/re-MINIAOD/ctpps_2018A_pixelefficiency_singleele200630_073503/0000/*.root | sed 's/^/file:/' > InputFiles/Era.dat```
+	3. **Set up config files** (.py files in python folder).  You just need make step 1 manually, all others values are correct. They are put here for debugging purposes.  
+		1. **MaxEvents** as test file is big just set maxEvents to smaller value ex. `100000`
+		2. **Global tag** Use `123X_dataRun2_v4` Global Tag in the python files.
+		3. **Process**  `eras.Run2_2018`
+		4. **Producer Tag** `CTPPSTestProtonReconstruction`
+		
+3. Run Efficiency analysis (referred as EA).
+	1. **Run EA (efficiency analysis) worker**

```

Comment:
Same as L74

Reply:
After moving the definition up - the acronym definition and its first usage doesn't fit on a single computer page, I will leave it for more clarity and for keeping the code neurodiverse accessible.

Reply:
Haha! Ok for me, no strong feelings. Only, it's good practice to give a definition the first time new terminology is introduced, then I don't mind if you remind the user about it later, it's just to make it readable!